### PR TITLE
Move metrics

### PR DESCRIPTION
**Pull request by kevinbarabash**
A first step towards #11.  This diff moves of all the history for the files in the `metrics` folder.

TODO:
- [x] create src/metrics folder
- [x] add a script to metrics generation pipeline and output file into src/metrics
- [x] run script and commit generated file

In order to maintain the `metrics` folder, I filtered out all other files from a clone of the KaTeX repo.  In that repo I ran:
```
git filter-branch --tree-filter "find . -type f -not -wholename './metrics/*' -delete" --prune-empty HEAD
```
That left me with the `metrics` folder and the `submodules/katex-fonts` folder.  To get rid of `submodules` I ran:
```
git filter-branch -f --prune-empty --tree-filter '
    git submodule deinit -f .
    git rm -rf submodules && rm -rf submodules
    find . -name .gitmodules -delete' HEAD
```
#10 has the rest of the details on maintaining history.

### CODE REVIEW

**Comment by ylemkimon** [comment]
It seems it includes empty merge commits, as `--prune-empty` `removes empty commits if they have exactly one or zero non-pruned parents; merge commits will therefore remain intact`.

You can either:
(i) use `parent-filter` after `filter-branch` (from [StackOverflow](https://stackoverflow.com/a/38420284))
```bash
git filter-branch --prune-empty --parent-filter 'sed "s/-p //g" | xargs -r git show-branch --independent | sed "s/\</-p /g"'
```
(ii) use `subdirectory-filter` and move the files to the subdirectory again
```bash
git filter-branch --prune-empty --subdirectory-filter metrics -- --all
mkdir metrics
mv * metrics
git add .
git commit
```

File: metrics/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  (possibly using the `cpan` command line tool) or with your package manager.
+
+- You need the python module fonttools. You can install this either from PyPi
+  (using `easy_install` or `pip`) or with your package manager.
+
+Once you have these things, run
+
+    make metrics
+
+which should generate new metrics and place them into `fontMetricsData.json`.
```
Comment (ylemkimon) [inline metrics/README.md]: Documentations need to be updated

**Comment by kevinbarabash** [comment]
> It seems it includes empty merge commits, as --prune-empty removes empty commits if they have exactly one or zero non-pruned parents; merge commits will therefore remain intact.

Interesting.

> (ii) use subdirectory-filter and move the files to the subdirectory again
>
> git filter-branch --prune-empty --subdirectory-filter metrics -- --all
> mkdir metrics
> mv * metrics
> git add .
> git commit

I tried the `--subdirectory-fitler` at first, but then it moved everything to the root folder and I had concerns about moving files around.  I guess it's not that big of a deal and it gives me an opportunity to move files into `src/metrics` which is where I actually wanted them.

**Comment by kevinbarabash** [comment]
Closing in preference of #30.

**Pull request closed without merging**